components/TextField/TextField.tsx

```diff
@@ -48,19 +51,11 @@ const TextField: FC<TextFieldTypeProps> = ({
         disabled={disabled}
         onChange={onChange}
       />
+      {variant === "error" && (
```

Comment:
question: What's the difference between the 4 types of `VariantType`?

What if the field is...
- standard and has error?
- filled and has an error?

What's the diff between "alert" and "error"? It would be nice a little code comment explaining the meaning of each variant. With that you might find out that there's a better organization :)

Reply:
not clear on this one but we can meet next time to talk about this